### PR TITLE
Remove AC from python demo

### PR DESCRIPTION
**Pull request by ZhmakaAS**
Removing AcraConnector from python demo and setup direct TLS communication with AcraServer.

Note: instead of static `ClientID`, now we are using dynamic generated from TLS certificate, so some example commands were updated with explicit defining `EXAMPLE_PUBLIC_KEY` which value we read from file with path to public key.

### CODE REVIEW

File: python/docker-compose.python.yml [hunk 1]
```diff
@@ -179,25 +116,24 @@ services:
...
-            ACRA_CLIENT_ID: ${ACRA_CLIENT_ID:-testclientid}
-            ACRA_CONNECTOR_HOST: acra-connector
-            ACRA_CONNECTOR_PORT: 9494
-            # Demo script default cofiguration
-            EXAMPLE_ACRA_CONNECTOR_API_ADDRESS: http://acra-connector:9191
-            EXAMPLE_HOST: acra-connector
-            EXAMPLE_PORT: 9494
-            EXAMPLE_DB_USER: ${POSTGRES_USER:-test}
+            # Demo script default configuration
+            EXAMPLE_ACRA_CONNECTOR_API_ADDRESS: http://acra-server:9090
```
Comment (vixentael) [inline python/docker-compose.python.yml:120]: should it be named `..acra_connector..`?

**Review by vixentael — APPROVED** [review]

File: python/docker-compose.python.yml (hunk 1 above)
Comment (ZhmakaAS) [reply python/docker-compose.python.yml:120]: Yes, need to fix it in acra examples firstly.

**Review by vixentael — APPROVED** [review]
👍

File: python/entry.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+find $PUBLIC_KEY_DIR -type f -name \*.pub -print > /pub_key_name.txt
```
Comment (shadinua) [inline python/entry.sh:3]:
```suggestion
find "$PUBLIC_KEY_DIR" -type f -name \*.pub -print > /pub_key_name.txt
```

**Review by shadinua — APPROVED** [review]

**Pull request merged**